features/Hotels/hotels.feature

```diff
@@ -0,0 +1,111 @@
+Feature: Project 1
+
+    @TC-18
+    Scenario: TC-18 Verify user can update number of guests on Homepage
+        Given I am on hotels landing page
+        When I click on Travelers
+        Then I select adults as "6"
+        Then I select children as "3"
+        Then I select first child age as 4
+        Then I select second child age under 1
+        Then I select third child age 7
+        When I click Done
+        Then I verify total number of guests in sum of adults and children same as selected on step 3 and 4
+
+    @TC-21
+    Scenario: TC-21 Verify verification message for invalid sign in credentials
```

Comment:
You can have Example table only with `Scenario Outline`.
Scenario => Scenario Outline